### ISSUE TITLE
github-parse-response: Symbol's function definition is void: json-read

### ISSUE THREAD

**Issue by JosefAssad** [inkel/github-issues.el#6]
I'm getting the above error when trying to run `github-issues`. In its entirety:
```
Contacting host: api.github.com:443
github-parse-response: Symbol's function definition is void: json-read
```
emacs version: GNU Emacs 24.4.1
github-issues version: 20120425.1535 (from package)

**Comment by inkel** [comment]
@JosefAssad thanks for reporting this, I'm sorry I didn't check it on time. It was fixed in #9.

**Comment by JosefMikkelsen** [comment]
Thank you @inkel!
